pygmt/base_plotting.py

```diff
@@ -303,20 +303,55 @@ def grdcontour(self, grid, **kwargs):
 
     @fmt_docstring
     @use_alias(
-        R="region",
-        J="projection",
-        W="pen",
+        A="img_out",
         B="frame",
-        I="shading",
         C="cmap",
+        D="img_in",
+        E="dpi",
+        G="bit_color",
```

Comment:
Aliases img_out (A), img_in (D) and bit_color (G) are simply following GMT.jl at https://www.generic-mapping-tools.org/GMT.jl/v0.22/#GMT.grdimage. Shout out if this is a good plan.

Alternative might be out_img (A) and color (G) to follow GMT doc page a little closer. See https://docs.generic-mapping-tools.org/6.1/grdimage.html#a.

Reply:
I like the GMT.jl names.

Reply:
Ok, stick with them then.